integrations/artifactory.rst

```diff
@@ -0,0 +1,57 @@
+.. _integrations_artifactory:
+
+Artifactory
+============
+
+`JFrog's Artifactory <https://jfrog.com/artifactory/>`_ is used to power `ConanCenter`_ and it is the single solution
+for housing and managing all the binaries, packages, and files for use throughout your software supply chain. Artifactory
```

Comment:
I thinks these 2 first paragraphs are missing very important points:
- It doesn't explicitly describe that it can host all package types (not formats), and cite "docker, maven, npm, ...., and of course Conan packages", in a **private** repository 
- It doesn't mention the main benefits of storing and managing binary packages: saving build time, better traceability, security, more robust builds... (instead of words like "software suply chain"). I would replace that with the benefits. 
- It only mention developers, but not CI, but the repos are critical and very useful for CI.
- I wouldn't cite bad known practices as shared drive or SCM tools as "possible" solutions, but as legacy or wrong solutions (not just limited)